packages/client/src/pages/CarbonHackCards/ForecastCard/ForecastLineChart.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import ReactApexChart from 'react-apexcharts'
+
+export class ForecastLineChart extends React.Component<
```

Comment:
This should be a functional component instead of a class component. There should be a different way to construct the chart config without the use of constructors, similar to how we do for the other components